file-browser/package.json

```diff
@@ -22,17 +22,19 @@
   },
   "private": true,
   "dependencies": {
+    "electron-is-dev": "^0.2.0",
+    "eywa": "git+https://github.com/escherpad/eywa.git",
```

Comment:
@jam-world On a side note, we might want to change eywa's name and publish it somewhere.